docs/dev/news/2907.feat

```diff
@@ -0,0 +1,3 @@
+Added support for Dafny project files. For now the project file only allows specifying which Dafny files to include and exclude, and what options to use.
```

Comment:
Should probably mention the supported format is `*.toml` here, or perhaps link to the refman section.